src/Qowaiv.Validation.DataAnnotations/AnnotatedProperty.cs

```diff
@@ -61,5 +61,11 @@ internal string DebuggerDisplay
     [Pure]
     internal static IEnumerable<AnnotatedProperty> CreateAll(Type type)
         => type.GetProperties()
+        .Where(Include)
         .Select(property => new AnnotatedProperty(property));
+
+    [Pure]
+    private static bool Include(PropertyInfo property)
+        => property.CanRead
+        && !property.GetIndexParameters().Any();
```

Comment:
Shouldn't an annotated property have an index? Or am I misunderstanding something?

Reply:
Parameters of this form:
``` C#
public int this[int index] => //...
```

Those should be skipped.